benchmark/single-source/StringComparison.swift.gyb

```diff
@@ -47,25 +47,18 @@ public let StringHashing = [
   BenchmarkInfo(
     name: "StringHashing_${Name}",
     runFunction: run_StringHashing_${Name},
-    tags: [.validation, .api, .String]),    
+    tags: [.validation, .api, .String],
+    setUpFunction: { blackHole(Workload_${Name}) }),
 % end # Names
 ]
 
 % for Name in Names:
 
-var Workload_${Name}: Workload? = nil
-
-@inline(never)
-public func setup_StringComparison_${Name}() {
-  if Workload_${Name} != nil {
-    return
-  }
-  Workload_${Name} = Workload.${Name}
-}
+var Workload_${Name}: Workload! = Workload.${Name}
```

Comment:
This whole file is absolutely beautiful. I haven't seen this benchmark before, as I was stuck on a November 2017 fork. This is exactly the correct pattern to emulate when you have a significant setup overhead. Excellent engineering all over here!